test/acceptance/authz/rbac_auto_viewer_permissions_test.go

```diff
@@ -65,7 +65,13 @@ func TestAuthzAllEndpointsViewerDynamically(t *testing.T) {
 		"/authz/roles/{id}/has-permission", // must be a POST rather than GET or HEAD due to need of body. but viewer can access it due to its permissions
 	}
 
+	// DynUser-Todo: remove when permissions are fixed
```

Comment:
Is this TODO still required?

Reply:
Yep - all endpoints have READ authorization right now, eg the viewer succeeds. However, POST should never work for a viewer and this test would fail. My next step is to add the relevant permissions and then remove this :)